subsys/net/ip/net_if.c

```diff
@@ -356,6 +356,12 @@ enum net_verdict net_if_send_data(struct net_if *iface, struct net_pkt *pkt)
 		goto done;
```

Comment:
The change is ok. Just minor nit for the commit message, there is now double `net: net_...` there which is quite pointless, could you change that to `net: if: ...`

Reply:
fixed